clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -495,6 +392,93 @@ SYCLToolChain::TranslateArgs(const llvm::opt::DerivedArgList &Args,
   return DAL;
 }
 
+// Expects a specific type of option (e.g. -Xsycl-target-backend) and will
+// extract the arguments.
+static void TranslateTargetOpt(const ToolChain &TC,
```

Comment:
I believe this could be a private method of `SYCLToolchain' to facilitate `this` usage (as per one of the previous comments)

Reply:
Done.